app/lib/dttp/params/placement_assignment.rb

```diff
@@ -18,6 +18,8 @@ class PlacementAssignment
 
       SCHOLARSHIP = "188375c2-7722-e711-80c8-0050568902d3"
 
+      NO_BURSARY_AWARDED = "1c8375c2-7722-e711-80c8-0050568902d3"
```

Comment:
I wonder whether these two constants (SCHOLARSHIP and NO_BURSARY_AWARDED) should be stored in the `Dttp::Codesets::BursaryDetails` module, where all the other entity IDs are.

Reply:
Good idea, I will add that in!